services/scabbard/src/cli/error.rs

```diff
@@ -0,0 +1,137 @@
+// Copyright 2020 Cargill Incorporated
```

Comment:
Needs to be // Copyright 2018-2020 Cargill Incorporated
(same for rest of the new files)

Reply:
Updated